src/controller/Controller.js

```diff
@@ -80,6 +81,13 @@ export class Controller {
     this.keys = [];
     for (let i = 0, l = keys.length; i < l; i++) {
       const currentKey = keys[i].key.toLowerCase();
+      const altKey = alternateKeyMap[currentKey];
+
+      if (altKey) {
```

Comment:
Let's do a stronger check here, rather than a truthy one. Perhaps:

```javascript
if (altKey !== undefined) {
  // ...
}
```

just to be explicit